code/game/objects/items/twohanded.dm

```diff
@@ -492,9 +492,10 @@
 
 /obj/item/twohanded/spear/afterattack(atom/movable/AM, mob/user, proximity)
 	. = ..()
+	
 	if(!proximity)
 		return
-	if(isopenturf(AM)) //So you can actually melee with it
+	if(!istype(src, /obj/item/twohanded/spear/explosive) && isopenturf(AM)) //So you can actually melee with it
 		return
```

Comment:
...these 2 if/return statements are doing nothing

Reply:
what did the openturf one do originally? i could never figure out